mix.exs

```diff
@@ -39,10 +39,11 @@ defmodule DocuSign.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:poison, "~> 5.0"},
+      {:tesla, "~> 1.4"},
+      {:poison, "~> 3.0"},
```

Comment:
Any reason why `poison` was downgraded from 5.0 to 3.0?

Reply:
I used the version in the new generated app because I believed at the time that it would be higher than the current one. Now I see that only "tesla" was upgraded (to version 1.4) and poison had a downgrade. I will change poison version back to 5.0.